src/svgDocument.ts

```diff
@@ -0,0 +1,128 @@
+import svgPathBbox from "svg-path-bbox";
+
+/**
+ * An instance of this object can pretend to be the global "document"
+ * variable used in a browser context so that the calls made by the display()
+ * function populate our structure instead of manipulating DOM nodes.
+ */
```

Comment:
So this is basically implementing the DOM functions that display.js uses, right? I wonder if there is some polyfill out there that would be more robust to use? But that might be really big. Honestly if this works (I haven't tested it) it seems like a nice solution.

Reply:
I don't know either, I just went for a quick and simple string-based implementation that worked.

I have pushed one more commit that introduces a CLI tool to use this function. I have used esbuild to produce a standalone bundle and added a separate script target in package.json for it to avoid overriding your default build.